File: ___HappyCityScripts/EginPlugins/Connect/ConnectDefine.cs
using UnityEngine;
using System.Collections;

public class ConnectDefine {
    //510k合并入
    public static WWW Connectwww;

	public static string HostURL = PlatformGameDefine.playform.HostURL;

	// Login
	public static string LOGIN_URL = PlatformGameDefine.playform.HostURL + "/client_unity/login/";
	public static string GUEST_LOGIN_URL = PlatformGameDefine.playform.HostURL + "/client_unity/guestlogin/";
	// Register
	public static string REGISTER_VERIFY_CODE_URL = PlatformGameDefine.playform.HostURL + "/client_unity/captcha/";
	public static string REGISTER_VERIFY_CODE_PREFIX_URL = PlatformGameDefine.playform.HostURL;
	//	public static string REGISTER_URL = PlatformGameDefine.playform.HostURL + "/unity/register/"; 	// shawn.debug
	public static string REGISTER_MOBILE_URL = PlatformGameDefine.playform.HostURL + "/client_unity/register_mobile/";//"/mobile/register/";
    public static string REGISTER_WEIXIN_URL = PlatformGameDefine.playform.HostURL + "/account/openid/wechat_oauth/";//微信登录

    public static string REGISTER_MOBILE_PHONE_URL = PlatformGameDefine.playform.HostURL + "/mobile/regphone/";
	public static string GUEST_PHONE_REGIST_URL = PlatformGameDefine.playform.HostURL + "/mobile/guestphonereg/";
	public static string GUEST_RESGISTER_URL = PlatformGameDefine.playform.HostURL + "/client_unity/guestregister/";
	// Userinfo
	public static string USERINFO_AVATAR_URL = PlatformGameDefine.playform.HostURL + "/client_unity/modifyavatar/";
	public static string USERINFO_PASSWORD_URL = PlatformGameDefine.playform.HostURL + "/client_unity/changepass/";
	public static string USERINFO_BANK_PASSWORD_URL = PlatformGameDefine.playform.HostURL + "/client_unity/changebankpass/";
	// Convert
	public static string GOLDINGOT_COIN_URL = PlatformGameDefine.playform.HostURL + "/client_unity/dollar2money/";
	public static string GOLDINGOT_GOLDCOIN_URL = PlatformGameDefine.playform.HostURL + "/client_unity/dollar2gold/";
	public static string GOLDCOIN_COIN_URL = PlatformGameDefine.playform.HostURL + "/client_unity/gold2money/";
	public static string GOLDCOIN_HAPPYCARD_URL = PlatformGameDefine.playform.HostURL + "/client_unity/gold2happy/";
	public static string HAPPYCARD_COIN_SUIT_URL = PlatformGameDefine.playform.HostURL + "/client_unity/happy2moneysuit/";
	public static string HAPPYCARD_COIN_URL = PlatformGameDefine.playform.HostURL + "/client_unity/happy2money/";

    // Bank510k
    public static string BANK_LOGON_URL = PlatformGameDefine.playform.HostURL + "/home/bank_logon_flash/";
    public static string BANK_LOGOUT_URL = PlatformGameDefine.playform.HostURL + "/home/bank_logout_flash/";
    public static string BANK_LOGONRECORD_URL = PlatformGameDefine.playform.HostURL + "/home/bank_logon_record_json/";
    public static string BANK_CHECKPWD_URL = PlatformGameDefine.playform.HostURL + "/home/bank_check_pwd_flash/";
    public static string BANK_VALIDATE_URL = PlatformGameDefine.playform.HostURL + "/home/logon_validate_flash/";
    public static string BANK_WELCOME_URL = PlatformGameDefine.playform.HostURL + "/home/bank_welcome_flash/";

	// Bank
	public static string BANK_SAVE_URL = PlatformGameDefine.playform.HostURL + "/client_unity/bag2bank/";
	public static string BANK_GET_URL = PlatformGameDefine.playform.HostURL + "/client_unity/bank2bag/";
	public static string BANK_RECORD_URL = PlatformGameDefine.playform.HostURL + "/client_unity/bank_record_json/";
	// GameRecord
	public static string GAME_RECORD_URL = PlatformGameDefine.playform.HostURL + "/client_unity/game_record/";
	// Backpack
	public static string BACKPACK_LIST_URL = PlatformGameDefine.playform.HostURL + "/client_unity/bag_list/";
	// Mail
	public static string MAIL_COUNT_URL = PlatformGameDefine.playform.HostURL + "/client_unity/mail/mailcount/";
	public static string MAIL_LIST_URL = PlatformGameDefine.playform.HostURL + "/client_unity/mail/maillist/";
	public static string MAIL_DETAIL_URL = PlatformGameDefine.playform.HostURL + "/client_unity/mail/";
	public static string MAIL_DELETE_URL = PlatformGameDefine.playform.HostURL + "/client_unity/mail/maildel/";
	public static string MAIL_SEND_URL = PlatformGameDefine.playform.HostURL + "/client_unity/mail/mailwrite/";
	// Link
	public static string FORGET_PASSWORD_URL = PlatformGameDefine.playform.HostURL + "/account/pwd_reset/";
	public static string HOME_URL = PlatformGameDefine.playform.HostURL;
	public static string SERVICE_URL = PlatformGameDefine.playform.HostURL + "/us/online.html";
	public static string RECHARGE_URL = PlatformGameDefine.playform.HostURL + "/account/payment/";
	public static string GUIDE_URL = PlatformGameDefine.playform.HostURL + "/help/center.html";
	public static string SECURITY_URL = PlatformGameDefine.playform.HostURL + "/client_home/security/";
	public static string SUGGEST_URL = PlatformGameDefine.playform.HostURL + "/us/suggest.html";
	public static string BACKPACK_PROP_URL = PlatformGameDefine.playform.HostURL + "/client_home/props/";
	public static string REGISTER_AGREEMENT_URL = PlatformGameDefine.playform.HostURL + "/terms.html";
	// Other
	public static string FEEDBACK_URL = PlatformGameDefine.playform.HostURL + "/open/bug/";
	public static string UPDATE_USERINFO_URL = PlatformGameDefine.playform.HostURL + "/client_unity/userinfo/";
	public static string ROOM_LIST_URL = PlatformGameDefine.playform.HostURL + "/client_unity/gamerooms/";
	
	/* ------ hh.cc mobile ------ */
	// Safe
	public static string SAFE_VALIDATE_TYPE_URL = PlatformGameDefine.playform.HostURL + "/client_unity/safe_validate/";
	public static string SAFE_SEND_PHONECODE_URL = PlatformGameDefine.playform.HostURL + "/client_unity/send_phone_code/";
	public static string SAFE_VALIDATE_LOGIN_URL = PlatformGameDefine.playform.HostURL + "/client_unity/bank_login/";
	// Award
	public static string AWARD_CHECK_URL = PlatformGameDefine.playform.HostURL + "/client_unity/device/reward/";
	public static string AWARD_GET_URL = PlatformGameDefine.playform.HostURL + "/client_unity/device/reward/";
	// Gift
	public static string GIFT_DELIVER_URL = PlatformGameDefine.playform.HostURL + "/client_unity/deliver_money/";
	public static string GIFT_NICKNAME_URL = PlatformGameDefine.playform.HostURL + "/client_unity/nickname_by_uid/";
	public static string GIFT_RECORD_URL = PlatformGameDefine.playform.HostURL + "/client_unity/deliver_record_json/";
	// Recharge
	public static string RECHARGE_CARD_SUIT_URL = PlatformGameDefine.playform.HostURL + "/client_unity/yeepaysuit/";
	public static string RECHARGE_OFFICIAL_CARD_URL = PlatformGameDefine.playform.HostURL + "/client_unity/cardpay/";
	public static string RECHARGE_CARD_URL = PlatformGameDefine.playform.HostURL + "/client_unity/yeepayexchange/";
	
	//UPDATE
	//	public static string VERSION_CHECK_URL = PlatformGameDefine.playform.HostURL + "/unity/config/";
	
	// bank 
	public static string BANK_RECHARGE_URL = PlatformGameDefine.playform.HostURL + "/client_unity/pay/";
    //微信支付请求后台地址
	public static string BANK_RECHARGE_WX_URL = PlatformGameDefine.playform.HostURL + "/client_unity/payment_confirm/";//
    //微信支付完成后查询支付结果地址
	public static string BANK_RECHARGE_WX_Confirm_URL = PlatformGameDefine.playform.HostURL + "/client_unity/wechatmobile/orderquery/";//

    // TIAN XIA FU
    public static string TXF_RECHARGE_URL = PlatformGameDefine.playform.HostURL + "/client_unity/tianxiapay/";
	
	// bank bili
	public static string BANK_BILI_URL = PlatformGameDefine.playform.HostURL + "/client_unity/payrateshengpay/";//
	
	/// ZFB bili
	public static string ZFB_BILI_URL = PlatformGameDefine.playform.HostURL + "/client_unity/payratealipay/";//
	
	// ka  bili
	public static string CARD_BILI_URL = PlatformGameDefine.playform.HostURL + "/client_unity/payratecardsuit/";//
	//verify
	public static string VERIFY_CODE_URL = PlatformGameDefine.playform.HostURL + "/client_home/safe_device/";
	
	//change agent
	public static string CHANGE_AGENT_URL = PlatformGameDefine.playform.HostURL + "/client_home/change_agent/";
	
	//yanzheng
	public static string YAN_ZHENG_URL = PlatformGameDefine.playform.HostURL + "/client_unity/captcha/";
    public static string CONNECTHOST = PlatformGameDefine.playform.HostURL;

	public static void updateConfig(){
		HostURL = PlatformGameDefine.playform.HostURL;
		// Login
		LOGIN_URL = PlatformGameDefine.playform.HostURL + "/client_unity/login/";
		GUEST_LOGIN_URL = PlatformGameDefine.playform.HostURL + "/client_unity/guestlogin/";
		// Register
		REGISTER_VERIFY_CODE_URL = PlatformGameDefine.playform.HostURL + "/client_unity/captcha/";
		REGISTER_VERIFY_CODE_PREFIX_URL = PlatformGameDefine.playform.HostURL;
		REGISTER_MOBILE_URL = PlatformGameDefine.playform.HostURL + "/client_unity/register_mobile/";//"/mobile/register/";
		
		REGISTER_MOBILE_PHONE_URL = PlatformGameDefine.playform.HostURL + "/mobile/regphone/";
		GUEST_PHONE_REGIST_URL = PlatformGameDefine.playform.HostURL + "/mobile/guestphonereg/";
		GUEST_RESGISTER_URL = PlatformGameDefine.playform.HostURL + "/client_unity/guestregister/";
		// Userinfo
		USERINFO_AVATAR_URL = PlatformGameDefine.playform.HostURL + "/client_unity/modifyavatar/";
		USERINFO_PASSWORD_URL = PlatformGameDefine.playform.HostURL + "/client_unity/changepass/";
		USERINFO_BANK_PASSWORD_URL = PlatformGameDefine.playform.HostURL + "/client_unity/changebankpass/";
		// Convert
		GOLDINGOT_COIN_URL = PlatformGameDefine.playform.HostURL + "/client_unity/dollar2money/";
		GOLDINGOT_GOLDCOIN_URL = PlatformGameDefine.playform.HostURL + "/client_unity/dollar2gold/";
		GOLDCOIN_COIN_URL = PlatformGameDefine.playform.HostURL + "/client_unity/gold2money/";
		GOLDCOIN_HAPPYCARD_URL = PlatformGameDefine.playform.HostURL + "/client_unity/gold2happy/";
		HAPPYCARD_COIN_SUIT_URL = PlatformGameDefine.playform.HostURL + "/client_unity/happy2moneysuit/";
		HAPPYCARD_COIN_URL = PlatformGameDefine.playform.HostURL + "/client_unity/happy2money/";
		// Bank
		BANK_SAVE_URL = PlatformGameDefine.playform.HostURL + "/client_unity/bag2bank/";
		BANK_GET_URL = PlatformGameDefine.playform.HostURL + "/client_unity/bank2bag/";
		BANK_RECORD_URL = PlatformGameDefine.playform.HostURL + "/client_unity/bank_record_json/";
		// GameRecord
		GAME_RECORD_URL = PlatformGameDefine.playform.HostURL + "/client_unity/game_record/";
		// Backpack
		BACKPACK_LIST_URL = PlatformGameDefine.playform.HostURL + "/client_unity/bag_list/";
		// Mail
		MAIL_COUNT_URL = PlatformGameDefine.playform.HostURL + "/client_unity/mail/mailcount/";
		MAIL_LIST_URL = PlatformGameDefine.playform.HostURL + "/client_unity/mail/maillist/";
		MAIL_DETAIL_URL = PlatformGameDefine.playform.HostURL + "/client_unity/mail/";
		MAIL_DELETE_URL = PlatformGameDefine.playform.HostURL + "/client_unity/mail/maildel/";
		MAIL_SEND_URL = PlatformGameDefine.playform.HostURL + "/client_unity/mail/mailwrite/";
		// Link
		FORGET_PASSWORD_URL = PlatformGameDefine.playform.HostURL + "/account/pwd_reset/";
		HOME_URL = PlatformGameDefine.playform.HostURL;
		SERVICE_URL = PlatformGameDefine.playform.HostURL + "/us/online.html";
		RECHARGE_URL = PlatformGameDefine.playform.HostURL + "/account/payment/";
		GUIDE_URL = PlatformGameDefine.playform.HostURL + "/help/center.html";
		SECURITY_URL = PlatformGameDefine.playform.HostURL + "/client_home/security/";
		SUGGEST_URL = PlatformGameDefine.playform.HostURL + "/us/suggest.html";
		BACKPACK_PROP_URL = PlatformGameDefine.playform.HostURL + "/client_home/props/";
		REGISTER_AGREEMENT_URL = PlatformGameDefine.playform.HostURL + "/terms.html";
		// Other
		FEEDBACK_URL = PlatformGameDefine.playform.HostURL + "/open/bug/";
		UPDATE_USERINFO_URL = PlatformGameDefine.playform.HostURL + "/client_unity/userinfo/";
		ROOM_LIST_URL = PlatformGameDefine.playform.HostURL + "/client_unity/gamerooms/";
		
		/* ------ hh.cc mobile ------ */
		// Safe
		SAFE_VALIDATE_TYPE_URL = PlatformGameDefine.playform.HostURL + "/client_unity/safe_validate/";
		SAFE_SEND_PHONECODE_URL = PlatformGameDefine.playform.HostURL + "/client_unity/send_phone_code/";
		SAFE_VALIDATE_LOGIN_URL = PlatformGameDefine.playform.HostURL + "/client_unity/bank_login/";
		// Award
		AWARD_CHECK_URL = PlatformGameDefine.playform.HostURL + "/client_unity/device/reward/";
		AWARD_GET_URL = PlatformGameDefine.playform.HostURL + "/client_unity/device/reward/";
		// Gift
		GIFT_DELIVER_URL = PlatformGameDefine.playform.HostURL + "/client_unity/deliver_money/";
		GIFT_NICKNAME_URL = PlatformGameDefine.playform.HostURL + "/client_unity/nickname_by_uid/";
		GIFT_RECORD_URL = PlatformGameDefine.playform.HostURL + "/client_unity/deliver_record_json/";
		// Recharge
		RECHARGE_CARD_SUIT_URL = PlatformGameDefine.playform.HostURL + "/client_unity/yeepaysuit/";
		RECHARGE_OFFICIAL_CARD_URL = PlatformGameDefine.playform.HostURL + "/client_unity/cardpay/";
		RECHARGE_CARD_URL = PlatformGameDefine.playform.HostURL + "/client_unity/yeepayexchange/";
		
		//UPDATE
		//	 VERSION_CHECK_URL = PlatformGameDefine.playform.HostURL + "/unity/config/";
		
		// bank 
		BANK_RECHARGE_URL = PlatformGameDefine.playform.HostURL + "/client_unity/pay/";

        //微信支付请求后台地址
        BANK_RECHARGE_WX_URL = PlatformGameDefine.playform.HostURL + "/client_unity/payment_confirm/";
        //微信支付完成后查询支付结果地址
        BANK_RECHARGE_WX_Confirm_URL = PlatformGameDefine.playform.HostURL + "/client_unity/wechatmobile/orderquery/";

        // TIAN XIA FU
        TXF_RECHARGE_URL = PlatformGameDefine.playform.HostURL + "/client_unity/tianxiapay/";
		
		// bank bili
		BANK_BILI_URL = PlatformGameDefine.playform.HostURL + "/client_unity/payrateshengpay/";
		
		/// ZFB bili
		ZFB_BILI_URL = PlatformGameDefine.playform.HostURL + "/client_unity/payratealipay/";
		
		// ka  bili
		CARD_BILI_URL = PlatformGameDefine.playform.HostURL + "/client_unity/payratecardsuit/";
		//verify
		VERIFY_CODE_URL = PlatformGameDefine.playform.HostURL + "/client_home/safe_device/";
		
		//change agent
		CHANGE_AGENT_URL = PlatformGameDefine.playform.HostURL + "/client_home/change_agent/";
		
		//yanzheng
		YAN_ZHENG_URL = PlatformGameDefine.playform.HostURL + "/client_unity/captcha/";
	}
	
}

File: Lua/Common/ConnectDefine.lua

 local this = LuaObject:New()
 ConnectDefine = this

	this.HostURL = PlatformGameDefine.playform.HostURL;
	-- Login
	this.LOGIN_URL = this.HostURL .. "/client_unity/login/";
	this.GUEST_LOGIN_URL = this.HostURL .. "/client_unity/guestlogin/";
	-- Register
	this.REGISTER_VERIFY_CODE_URL = this.HostURL .. "/client_unity/captcha/";
	this.REGISTER_VERIFY_CODE_PREFIX_URL = this.HostURL;
	--	this.REGISTER_URL = this.HostURL .. "/unity/register/"; 	-- shawn.debug
	this.REGISTER_MOBILE_URL = this.HostURL .. "/client_unity/register_mobile/";--"/mobile/register/";
    this.REGISTER_WEIXIN_URL = this.HostURL .. "/account/openid/wechat_oauth/";--微信登录

    this.REGISTER_MOBILE_PHONE_URL = this.HostURL .. "/mobile/regphone/";
	this.GUEST_PHONE_REGIST_URL = this.HostURL .. "/mobile/guestphonereg/";
	this.GUEST_RESGISTER_URL = this.HostURL .. "/client_unity/guestregister/";
	-- Userinfo
	this.USERINFO_AVATAR_URL = this.HostURL .. "/client_unity/modifyavatar/";
	this.USERINFO_PASSWORD_URL = this.HostURL .. "/client_unity/changepass/";
	this.USERINFO_BANK_PASSWORD_URL = this.HostURL .. "/client_unity/changebankpass/";
	-- Convert
	this.GOLDINGOT_COIN_URL = this.HostURL .. "/client_unity/dollar2money/";
	this.GOLDINGOT_GOLDCOIN_URL = this.HostURL .. "/client_unity/dollar2gold/";
	this.GOLDCOIN_COIN_URL = this.HostURL .. "/client_unity/gold2money/";
	this.GOLDCOIN_HAPPYCARD_URL = this.HostURL .. "/client_unity/gold2happy/";
	this.HAPPYCARD_COIN_SUIT_URL = this.HostURL .. "/client_unity/happy2moneysuit/";
	this.HAPPYCARD_COIN_URL = this.HostURL .. "/client_unity/happy2money/";
	-- Bank
	this.BANK_SAVE_URL = this.HostURL .. "/client_unity/bag2bank/";
	this.BANK_GET_URL = this.HostURL .. "/client_unity/bank2bag/";
	this.BANK_RECORD_URL = this.HostURL .. "/client_unity/bank_record_json/";
	-- GameRecord
	this.GAME_RECORD_URL = this.HostURL .. "/client_unity/game_record/";
	-- Backpack
	this.BACKPACK_LIST_URL = this.HostURL .. "/client_unity/bag_list/";
	-- Mail
	this.MAIL_COUNT_URL = this.HostURL .. "/client_unity/mail/mailcount/";
	this.MAIL_LIST_URL = this.HostURL .. "/client_unity/mail/maillist/";
	this.MAIL_DETAIL_URL = this.HostURL .. "/client_unity/mail/";
	this.MAIL_DELETE_URL = this.HostURL .. "/client_unity/mail/maildel/";
	this.MAIL_SEND_URL = this.HostURL .. "/client_unity/mail/mailwrite/";
	-- Link
	this.FORGET_PASSWORD_URL = this.HostURL .. "/account/pwd_reset/";
	this.HOME_URL = this.HostURL;
	this.SERVICE_URL = this.HostURL .. "/us/online.html";
	this.RECHARGE_URL = this.HostURL .. "/account/payment/";
	this.GUIDE_URL = this.HostURL .. "/help/center.html";
	this.SECURITY_URL = this.HostURL .. "/client_home/security/";
	this.SUGGEST_URL = this.HostURL .. "/us/suggest.html";
	this.BACKPACK_PROP_URL = this.HostURL .. "/client_home/props/";
	this.REGISTER_AGREEMENT_URL = this.HostURL .. "/terms.html";
	-- Other
	this.FEEDBACK_URL = this.HostURL .. "/open/bug/";
	this.UPDATE_USERINFO_URL = this.HostURL .. "/client_unity/userinfo/";
	this.ROOM_LIST_URL = this.HostURL .. "/client_unity/gamerooms/";
	
	-- ------ hh.cc mobile ------ */
	-- Safe
	this.SAFE_VALIDATE_TYPE_URL = this.HostURL .. "/client_unity/safe_validate/";
	this.SAFE_SEND_PHONECODE_URL = this.HostURL .. "/client_unity/send_phone_code/";
	this.SAFE_VALIDATE_LOGIN_URL = this.HostURL .. "/client_unity/bank_login/";
	-- Award
	this.AWARD_CHECK_URL = this.HostURL .. "/client_unity/device/reward/";
	this.AWARD_GET_URL = this.HostURL .. "/client_unity/device/reward/";
	-- Gift
	this.GIFT_DELIVER_URL = this.HostURL .. "/client_unity/deliver_money/";
	this.GIFT_NICKNAME_URL = this.HostURL .. "/client_unity/nickname_by_uid/";
	this.GIFT_RECORD_URL = this.HostURL .. "/client_unity/deliver_record_json/";
	-- Recharge
	this.RECHARGE_CARD_SUIT_URL = this.HostURL .. "/client_unity/yeepaysuit/";
	this.RECHARGE_OFFICIAL_CARD_URL = this.HostURL .. "/client_unity/cardpay/";
	this.RECHARGE_CARD_URL = this.HostURL .. "/client_unity/yeepayexchange/";
	
	--UPDATE
	--	this.VERSION_CHECK_URL = this.HostURL .. "/unity/config/";
	
	-- bank 
	this.BANK_RECHARGE_URL = this.HostURL .. "/client_unity/pay/";
    --微信支付请求后台地址
	this.BANK_RECHARGE_WX_URL = this.HostURL .. "/client_unity/payment_confirm/";--
    --微信支付完成后查询支付结果地址
	this.BANK_RECHARGE_WX_Confirm_URL = this.HostURL .. "/client_unity/wechatmobile/orderquery/";--
	  
    --支付宝支付请求后台地址
	this.BANK_RECHARGE_Ali_URL = this.HostURL .. "/unity/payment_confirm/";--
    --支付宝支付完成后查询支付结果地址
	this.BANK_RECHARGE_Ali_Confirm_URL = this.HostURL .. "/account/alipayappreturn/";--

    -- TIAN XIA FU
    this.TXF_RECHARGE_URL = this.HostURL .. "/client_unity/tianxiapay/";
	
	-- bank bili
	this.BANK_BILI_URL = this.HostURL .. "/client_unity/payrateshengpay/";--
	
	-- ZFB bili
	this.ZFB_BILI_URL = this.HostURL .. "/client_unity/payratealipay/";--
	
	-- ka  bili
	this.CARD_BILI_URL = this.HostURL .. "/client_unity/payratecardsuit/";--
	--verify
	this.VERIFY_CODE_URL = this.HostURL .. "/client_home/safe_device/";
	
	--change agent
	this.CHANGE_AGENT_URL = this.HostURL .. "/client_home/change_agent/";
	
	--yanzheng
	this.YAN_ZHENG_URL = this.HostURL .. "/client_unity/captcha/";
	
	function this:updateConfig()
	 	-- print('========update config ========'..this.HostURL)
		this.HostURL =   PlatformGameDefine.playform.HostURL
		-- Login
		this.LOGIN_URL = this.HostURL .. "/client_unity/login/";
		this.GUEST_LOGIN_URL = this.HostURL .. "/client_unity/guestlogin/";
		-- Register
		this.REGISTER_VERIFY_CODE_URL = this.HostURL .. "/client_unity/captcha/";
		this.REGISTER_VERIFY_CODE_PREFIX_URL = this.HostURL;
		this.REGISTER_MOBILE_URL = this.HostURL .. "/client_unity/register_mobile/";--"/mobile/register/";
		
		this.REGISTER_MOBILE_PHONE_URL = this.HostURL .. "/mobile/regphone/";
		this.GUEST_PHONE_REGIST_URL = this.HostURL .. "/mobile/guestphonereg/";
		this.GUEST_RESGISTER_URL = this.HostURL .. "/client_unity/guestregister/";
		-- Userinfo
		this.USERINFO_AVATAR_URL = this.HostURL .. "/client_unity/modifyavatar/";
		this.USERINFO_PASSWORD_URL = this.HostURL .. "/client_unity/changepass/";
		this.USERINFO_BANK_PASSWORD_URL = this.HostURL .. "/client_unity/changebankpass/";
		-- Convert
		this.GOLDINGOT_COIN_URL = this.HostURL .. "/client_unity/dollar2money/";
		this.GOLDINGOT_GOLDCOIN_URL = this.HostURL .. "/client_unity/dollar2gold/";
		this.GOLDCOIN_COIN_URL = this.HostURL .. "/client_unity/gold2money/";
		this.GOLDCOIN_HAPPYCARD_URL = this.HostURL .. "/client_unity/gold2happy/";
		this.HAPPYCARD_COIN_SUIT_URL = this.HostURL .. "/client_unity/happy2moneysuit/";
		this.HAPPYCARD_COIN_URL = this.HostURL .. "/client_unity/happy2money/";
		-- Bank
		this.BANK_SAVE_URL = this.HostURL .. "/client_unity/bag2bank/";
		this.BANK_GET_URL = this.HostURL .. "/client_unity/bank2bag/";
		this.BANK_RECORD_URL = this.HostURL .. "/client_unity/bank_record_json/";
		-- GameRecord
		this.GAME_RECORD_URL = this.HostURL .. "/client_unity/game_record/";
		-- Backpack
		this.BACKPACK_LIST_URL = this.HostURL .. "/client_unity/bag_list/";
		-- Mail
		this.MAIL_COUNT_URL = this.HostURL .. "/client_unity/mail/mailcount/";
		this.MAIL_LIST_URL = this.HostURL .. "/client_unity/mail/maillist/";
		this.MAIL_DETAIL_URL = this.HostURL .. "/client_unity/mail/";
		this.MAIL_DELETE_URL = this.HostURL .. "/client_unity/mail/maildel/";
		this.MAIL_SEND_URL = this.HostURL .. "/client_unity/mail/mailwrite/";
		-- Link
		this.FORGET_PASSWORD_URL = this.HostURL .. "/account/pwd_reset/";
		this.HOME_URL = this.HostURL;
		this.SERVICE_URL = this.HostURL .. "/us/online.html";
		this.RECHARGE_URL = this.HostURL .. "/account/payment/";
		this.GUIDE_URL = this.HostURL .. "/help/center.html";
		this.SECURITY_URL = this.HostURL .. "/client_home/security/";
		this.SUGGEST_URL = this.HostURL .. "/us/suggest.html";
		this.BACKPACK_PROP_URL = this.HostURL .. "/client_home/props/";
		this.REGISTER_AGREEMENT_URL = this.HostURL .. "/terms.html";
		-- Other
		this.FEEDBACK_URL = this.HostURL .. "/open/bug/";
		this.UPDATE_USERINFO_URL = this.HostURL .. "/client_unity/userinfo/";
		this.ROOM_LIST_URL = this.HostURL .. "/client_unity/gamerooms/";
		
		-------- hh.cc mobile ------ */
		-- Safe
		this.SAFE_VALIDATE_TYPE_URL = this.HostURL .. "/client_unity/safe_validate/";
		this.SAFE_SEND_PHONECODE_URL = this.HostURL .. "/client_unity/send_phone_code/";
		this.SAFE_VALIDATE_LOGIN_URL = this.HostURL .. "/client_unity/bank_login/";
		-- Award
		this.AWARD_CHECK_URL = this.HostURL .. "/client_unity/device/reward/";
		this.AWARD_GET_URL = this.HostURL .. "/client_unity/device/reward/";
		-- Gift
		this.GIFT_DELIVER_URL = this.HostURL .. "/client_unity/deliver_money/";
		this.GIFT_NICKNAME_URL = this.HostURL .. "/client_unity/nickname_by_uid/";
		this.GIFT_RECORD_URL = this.HostURL .. "/client_unity/deliver_record_json/";
		-- Recharge
		this.RECHARGE_CARD_SUIT_URL = this.HostURL .. "/client_unity/yeepaysuit/";
		this.RECHARGE_OFFICIAL_CARD_URL = this.HostURL .. "/client_unity/cardpay/";
		this.RECHARGE_CARD_URL = this.HostURL .. "/client_unity/yeepayexchange/";
		
		--UPDATE
		--	 this.VERSION_CHECK_URL = this.HostURL .. "/unity/config/";
		
		-- bank 
		this.BANK_RECHARGE_URL = this.HostURL .. "/client_unity/pay/";

        --微信支付请求后台地址
        this.BANK_RECHARGE_WX_URL = this.HostURL .. "/client_unity/payment_confirm/";
        --微信支付完成后查询支付结果地址
        this.BANK_RECHARGE_WX_Confirm_URL = this.HostURL .. "/client_unity/wechatmobile/orderquery/";

        -- TIAN XIA FU
        this.TXF_RECHARGE_URL = this.HostURL .. "/client_unity/tianxiapay/";
		
		-- bank bili
		this.BANK_BILI_URL = this.HostURL .. "/client_unity/payrateshengpay/";
		
		-- ZFB bili
		this.ZFB_BILI_URL = this.HostURL .. "/client_unity/payratealipay/";
		
		-- ka  bili
		this.CARD_BILI_URL = this.HostURL .. "/client_unity/payratecardsuit/";
		--verify
		this.VERIFY_CODE_URL = this.HostURL .. "/client_home/safe_device/";
		
		--change agent
		this.CHANGE_AGENT_URL = this.HostURL .. "/client_home/change_agent/";
		
		--yanzheng
		this.YAN_ZHENG_URL = this.HostURL .. "/client_unity/captcha/";
	end
	
